### PR TITLE
CFP-28750: Multicast support in Cilium

### PR DESCRIPTION
**Pull request by hacktivist123**
This PR stores [CFP-2023-10-24: multicast support in Cilium](https://docs.google.com/document/d/14ekQd1MoKK9_NX0ru9rrhIHSfOyk_WkKQF7-aTLt3I0/edit?usp=sharing) in git for future reference.

Author of the CFP: @lliu0947

### CODE REVIEW

**Comment by joestringer** [comment]
cc @ldelossa does this match the latest implementation details?

File: cilium/CFP-2023-10-24-multicast-support-in-cilium.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+# CFP-2023-10-24: multicast support in cilium
```
Comment (joestringer) [inline cilium/CFP-2023-10-24-multicast-support-in-cilium.md:1]: https://github.com/cilium/cilium/issues/28750
```suggestion
# CFP-28750: multicast support in cilium
```

File: cilium/CFP-2023-10-24-multicast-support-in-cilium.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+
+# Meta
+
+**SIG: SIG-AGENT**
```
Comment (joestringer) [inline cilium/CFP-2023-10-24-multicast-support-in-cilium.md:6]:
```suggestion
**SIG: SIG-Datapath**
```

**Review by joestringer — CHANGES_REQUESTED** [review]

**Comment by xmulligan** [comment]
@lliu0947 do you mind if I close this since multicast has been implemented separately from this CFP in Cilium?

**Comment by joestringer** [comment]
As much as I would like to have the design of multicast documented, at this point that's up to the contributors involved in that effort to do the write-up. I don't think it's helping the situation if the three people interacting on this PR aren't involved in the proposal.

**Pull request closed without merging**